tests/QtQuick/Rectangle.js

```diff
@@ -1,23 +1,23 @@
 describe('QtQuick.Rectangle', function() {
+  setupDivElement();
   var loader = prefixedQmlLoader('QtQuick/qml/Rectangle');
+
   it('White', function() {
-    var div = loader('White');
+    var qml = loader('White', this.div);
+    var div = this.div;
```

Comment:
Inconsistency between here and `Render/runner.js`. Also, `qml` in unused here.
Perhaps just `var div = loadQmlFile(test.qml, this.div).dom;`?